Source/HedgeGI/BakingFactoryWindow.cpp

```diff
@@ -66,6 +66,10 @@ const Label OPTIMIZE_SEAMS_LABEL = { "Optimize Seams",
 const Label SKIP_EXISTING_FILES_LABEL = { "Skip Existing Files",
     "Skips instances that already have their resulting images in the output directory." };
 
+const Label SAVE_AS_BC7_LABEL = { "Save As BC7",
+    "Saves lightmap atlases using the higher-quality BC7 compression.\n\n" 
+    "This will only work in games that support handling said format, such as Unleashed Recompiled or Generations with the D3D11 mod." };
```

Comment:
```suggestion
    "This will only work in games that support handling said format, such as Unleashed Recompiled or Sonic Generations with the D3D11 mod." };
```